helm/g2p-sandbox/config/application-bb.properties

```diff
@@ -1,6 +1,6 @@
 store.local.interop.host=https://rhino.mifos.g2pconnect.io
 store.local.customer.host=https://rhino.mifos.g2pconnect.io
-zeebe.broker.contactpoint=zeebe-zeebe-gateway:26500
+zeebe.broker.contactpoint={{ .Release.Name }}-zeebe-gateway:26500
```

Comment:
you had confirmed that this change doesn't work in the configuration file.  Did you find a way to make this work?